agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -33,30 +33,28 @@ using namespace beerocks::net;
 /////////////////////////// Local Module Functions ///////////////////////////
 //////////////////////////////////////////////////////////////////////////////
 
-static void copy_radio_supported_channels(std::shared_ptr<bwl::ap_wlan_hal> &ap_wlan_hal,
-                                          beerocks::message::sWifiChannel supported_channels[])
+static void copy_radio_channels(std::vector<bwl::WiFiChannel> radio_channels,
```

Comment:
I think a better name now should be just `copy_channels` cause thats all it does:
```
static void copy_channels(std::vector<bwl::WiFiChannel> &from,
                                         beerocks::message::sWiFiChannel to[]);
```
(nitpick)